spec/support/shared_examples/sign_in.rb

```diff
@@ -95,7 +94,7 @@
     end
   end
 
-  it 'gets bad password error', :email, js: true do
+  it 'gets bad password error' do
```

Comment:
was this spec testing the bad password error in JS as well?

Reply:
Nope: https://github.com/18F/identity-idp/blob/0ba95508370c30aa34ff9960fac9f3493c2030bd/spec/support/shared_examples/sign_in.rb#L396-L416